tests/integration/gateway_clients/test_streaming.py

```diff
@@ -93,7 +93,7 @@ async def _compute_response():
 
         async def task_wrapper():
             response_msg = await _compute_response()
-            return [response_msg]
```

Comment:
Was this failing?

Reply:
yes, the interface is wrong, a call returns only a single message. I think that changed at some point in the last weeks and was not changed here.